--- xinitrc-ng-1.1/prefdm~	2012-04-27 21:26:14.000000000 +0300
+++ xinitrc-ng-1.1/prefdm	2013-09-06 00:22:55.596685047 +0300
@@ -16,6 +16,8 @@
 		preferred=/usr/bin/kdm
 	elif [ "$DISPLAYMANAGER" = LightDM ]; then
 		preferred=/usr/sbin/lightdm
+		# reset argv, lightdm does not like "-nodaemon" argument
+		set --
 	elif [ "$DISPLAYMANAGER" = LXDE ]; then
 		preferred=/usr/sbin/lxdm
 	elif [ "$DISPLAYMANAGER" = WDM ]; then
