--- xinitrc-ng-1.1/prefdm.orig	2012-04-27 20:26:14.000000000 +0200
+++ xinitrc-ng-1.1/prefdm	2021-02-18 13:31:01.084927767 +0100
@@ -22,6 +22,8 @@
 		preferred=/usr/bin/wdm
 	elif [ "$DISPLAYMANAGER" = XDM ]; then
 		preferred=/usr/bin/xdm
+	elif [ "$DISPLAYMANAGER" = SDDM ]; then
+		preferred=/usr/bin/sddm
 	elif [ -n "$DISPLAYMANAGER" ]; then
 		preferred=$DISPLAYMANAGER
 	fi
@@ -39,6 +41,7 @@
 [ -x /usr/sbin/gdm ] && exec /usr/sbin/gdm "$@" >/dev/null 2>&1 </dev/null
 [ -x /usr/bin/kdm ] && exec /usr/bin/kdm "$@" >/dev/null 2>&1 </dev/null
 [ -x /usr/bin/wdm ] && exec /usr/bin/wdm "$@" >/dev/null 2>&1 </dev/null
+[ -x /usr/bin/sddm ] && exec /usr/bin/sddm "$@" >/dev/null 2>&1 </dev/null
 
 # catch all exit error
 exit 1
